composer.json

```diff
@@ -14,7 +14,10 @@
         }
     },
     "require": {
-        "php": "^7.1"
+        "php": "^7.1",
+        "aerospike/aerospike-client-php": "~7.0",
+        "doctrine/cache": "^1.8",
+        "symfony/cache": "^4.2"
     },
```

Comment:
Also "provide" block should be added with "psr/simple-cache-implementation" and "provide psr/cache-implementation"

Reply:
Added